docs/operating/transaction-gas.md

```diff
@@ -17,7 +17,7 @@ Operators:
 - [Can set gas to zero for testing](#zero-gas-price-for-testing)
 - Must configure the Proxy to accept no-fee transactions
 
-<!-- Is this last item valid? Operators need to accept request to provide gasless (0 gas fee transaction). Is this handled by the Proxy without any configuration? -->
+<!-- wip Support needed for last item>> Operators need to accept request to provide no-fee transaction. Is this handled by the Proxy without any configuration? -->
```

Comment:
this is with @afalaleev

Reply:
@himaster

Reply:
NB to @m4sterbunny remember to remove ref to gasless and replace with fee free

Reply:
@BoogVAr If the BA team have completed documentation of the Proxy, I may be able to figure out the answers to some of these outstanding Qs myself. Alternatively, can someone from DevOps support? This had a ticket on Product/Docs board (as per the branch name), but this has been closed for the migration and become a Docs/Notion ticket https://www.notion.so/neonfoundation/Proxy-Operator-Docs-d459fbef226a44e08872c7a4b01b4335?pvs=4